pkg/apis/extensions/validation/validation.go

```diff
@@ -655,6 +656,33 @@ func validatePodSecurityPolicyVolumes(fldPath *field.Path, volumes []extensions.
 	return allErrs
 }
 
+const sysctlPatternSegmentFmt string = "([a-z0-9][_a-z0-9]*)?[a-z0-9*]"
```

Comment:
Can't you reuse ``SysctlSegmentFmt`` and ``SysctlFmt`` from ``pkg/api/validation/validation.go``?